packages/frontend/app/(dashboard)/course/[cid]/async_centre/components/modals/CreateAsyncQuestionModal.tsx

```diff
@@ -279,6 +301,32 @@ const CreateAsyncQuestionModal: React.FC<CreateAsyncQuestionModalProps> = ({
           <QuestionTagSelector questionTags={questionTypes} />
         </Form.Item>
       )}
+      <Form.Item
+        name="setAnonymous"
+        label="Hide name and avatar from other users?"
```

Comment:
```suggestion
        label="Hide name and avatar from other users?"
        layout="horizontal"
```
I would add this to both of these switches